Makefile

```diff
@@ -18,25 +18,34 @@ sdcard: uefi_release
 	parted -s sdcard.img unit s mkpart uboot 8MiB 16MiB
 	parted -s sdcard.img unit s mkpart env 16MiB 32MiB
 
-	for size in 4GB 8GB; do						\
-		cp sdcard.img sdcard_$${size}.img;			\
-		dd if=idblock.bin of=sdcard_$${size}.img 		\
-		    seek=64 conv=notrunc;				\
-		dd if=QUARTZ64_EFI_$${size}.itb of=sdcard_$${size}.img	\
-		    seek=20480 conv=notrunc;				\
+	# Quartz64 boards
+	for size in 4GB 8GB; do							\
+		cp sdcard.img sdcard_Quartz64_$${size}.img;			\
+		dd if=idblock.bin of=sdcard_Quartz64_$${size}.img 		\
+		    seek=64 conv=notrunc;					\
+		dd if=Quartz64_EFI_$${size}.itb of=sdcard_Quartz64_$${size}.img	\
+		    seek=20480 conv=notrunc;					\
+	done
+	# SOQuartz modules
+	for size in 2GB 4GB 8GB; do						\
+		cp sdcard.img sdcard_SOQuartz_$${size}.img;			\
+		dd if=idblock.bin of=sdcard_SOQuartz_$${size}.img 		\
+		    seek=64 conv=notrunc;					\
+		dd if=SOQuartz_EFI_$${size}.itb of=sdcard_SOQuartz_$${size}.img	\
+		    seek=20480 conv=notrunc;					\
 	done
 	rm -f sdcard.img
 
 .PHONY: release
 release: sdcard
-	gzip sdcard_4GB.img
-	gzip sdcard_8GB.img
+	gzip sdcard_*_?GB.img
 
 .PHONY: clean
 clean:
 	rm -rf Build
 	rm -f bl31_*.bin
 	rm -f idblock.bin
-	rm -f QUARTZ64_EFI_*.itb
+	rm -f SOQuartz_EFI_*.itb
+	rm -f Quartz64_EFI_*.itb
```

Comment:
Can you keep these uppercase please?

Reply:
Just the *.itb files or do you want uppercase in the sdcard_* files also?

Reply:
Only really concerned about the itb files. No opinion on `sdcard_*` so do whatever is easiest there.

Reply:
In `build_fit()` we need the correct case for `Build/${board}` but we can use uppercase (`${board_upper}` perhaps) for 
`${board_upper}_EFI_*.itb`
`uefi_${board_upper}.its`
`sdcard_${board_upper}_*.img`

Reply:
Ok, I adjusted the Makefile & build.sh to uppercase the `*.itb` files only.